webpack.server.config.js

```diff
@@ -2,6 +2,7 @@ const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
+  mode: 'development',
```

Comment:
This was the problem why `ssr` would throw an error at run time, it defaulted to `production`.

Reply:
Oh really? Why did it work before then? What's different?